scripts/crontab/crontab.tpl

```diff
@@ -46,6 +46,9 @@ HOME=/tmp
 30 11 * * * %(z_cron)s update_addon_average_daily_users
 00 12 * * * %(z_cron)s index_latest_stats
 
+# Once per week
+0 12 * * 1 %(django)s review_reports
```

Comment:
Can you move that to 9:01am UTC ? Less risk of conflict with the stats cron, and by doing it 1 minute after the hour, it should run after the `auto_approve` cron as well just in case.